pkg/meta/meta.go

```diff
@@ -263,3 +329,31 @@ func ParseCompVersion(spec string) (string, v0manifest.Version) {
 	}
 	return spec, ""
 }
+
+// IsSupportedComponent return true if support if platform support the component.
+func (env *Environment) IsSupportedComponent(component string) bool {
+	if env.v1Repo != nil {
+		// TODO
+		return true
```

Comment:
Check where does the `index.json` contains the component name.

Reply:
component.json, but not index.json
if index.json does not contain the component, then means no this component